pkgs/top-level/go-packages.nix

```diff
@@ -2416,13 +2429,26 @@ let
     rev = "b0c168ac0cf9493da1f9bb76c34b26ffef940b4a";
     name = "yaml-v1-${stdenv.lib.strings.substring 0 7 rev}";
     goPackagePath = "gopkg.in/yaml.v1";
-    src = fetchgit {
+    src = fetchFromGitHub {
       inherit rev;
-      url = "https://github.com/go-yaml/yaml.git";
+      owner = "go-yaml";
+      repo = "yaml";
       sha256 = "0jbdy41pplf2d1j24qwr8gc5qsig6ai5ch8rwgvg72kq9q0901cy";
     };
   };
 
+  yaml-v2 = buildGoPackage rec {
+    rev = "c1cd2254a6dd314c9d73c338c12688c9325d85c6";
+    name = "yaml-v2-${stdenv.lib.strings.substring 0 7 rev}";
```

Comment:
I was just matching what nearly every other package in that file does, on the assumption that internal consistency was more important than strictly following the style guide.  I'll happily change it to a date string if you prefer, though.